src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -247,9 +247,10 @@ export default class EnhancedReactTable<T> extends React.Component<
 
         // always use the initially sorted columns (this.sortedColumns),
         // otherwise already hidden columns will never appear in the dropdown menu!
-        let columnVisibility: Array<
```

Comment:
[Keep type parameters inline for type annotations in variable declarations](https://prettier.io/blog/2019/11/09/1.19.0.html#keep-type-parameters-inline-for-type-annotations-in-variable-declarations-6467-by-sosukesuzuki)